plugins/woo-ai/src/image-background-removal/image-background-removal.scss

```diff
@@ -14,3 +14,7 @@
 	}
 
 }
+
+body.post-type-product .woocommerce-layout__footer {
```

Comment:
Not thrilled with this, although it's the only way to get the snackbar to display above the modal. If we don't do this, we'd have to come up with another UI approach to gather feedback. I didn't notice any issues with it but there's always a chance for some weirdness.